lib/utils/index.js

```diff
@@ -0,0 +1,11 @@
+export const debounce = (func) => {
```

Comment:
I like that you created your own `debounce` function instead of using `lodash.debounce`. Good job.